ProteoformSuiteGUI/AggregatedProteoforms.cs

```diff
@@ -82,13 +141,5 @@ private void nUD_Missed_Ks_ValueChanged(object sender, EventArgs e)
             Lollipop.aggregate_proteoforms();
         }
 
-        private void dgv_AcceptNeuCdLtProteoforms_CellContentClick(object sender, DataGridViewCellEventArgs e)
```

Comment:
why delete this? I think we need it to know which cell was clicked so that the accept/notaccept boolean can be toggled.